src/SFML/Graphics/View.cpp

```diff
@@ -46,11 +47,11 @@ m_invTransformUpdated(false)
 
 ////////////////////////////////////////////////////////////
 View::View(const FloatRect& rectangle) :
-m_center             (),
-m_size               (),
-m_rotation           (),
-m_viewport           ({0, 0}, {1, 1}),
-m_transformUpdated   (false),
+m_center(),
+m_size(),
+m_rotation(),
+m_viewport({0, 0}, {1, 1}),
+m_transformUpdated(false),
 m_invTransformUpdated(false)
```

Comment:
Unfortunately it doesn't look like there's an option to align constructor initializations, at least I haven't found one. But I think we can live with that 🙂

Reply:
Sadly no, as far as I'm aware. Luckily tons of these initializers can be removed (anything default initialized) or moved to the class declaration like I've started doing in #2156.